### ISSUE TITLE
Sample Django Benchmark

### ISSUE THREAD

**Issue by mrigankpawagi** [utahplt/static-python-perf#15]
The instructions are [here](https://github.com/utahplt/static-python-perf/tree/main/Benchmark/django_sample). It simply renders a graphic at the designated endpoint (`/myapp`) - taking inspiration from the django benchmarks in pyperformance and skybison. However, we are using `locust` for running the benchmark since this is also the tool we will use going forward as we create bigger benchmarks.

**Comment by bennn** [comment]
Which files might we add types to?

**Comment by mrigankpawagi** [comment]
No such files.

**Comment by bennn** [comment]
No not many at all!

Would it make sense to type parts of Django? Wherever the core work is happening is what we should aim for ... and hopefully that's fewer than 10 files.

**Comment by mrigankpawagi** [comment]
Yes, that would also be interesting... taking a simple django program and typing django itself (vs taking a complicated django program but using it with untyped django)
